afkak/consumer.py

```diff
@@ -25,14 +25,14 @@
 from twisted.internet.task import LoopingCall
 from twisted.python.failure import Failure
 
+from afkak._util import _coerce_consumer_group, _coerce_topic
```

Comment:
Why the different form ('afkak._util' vs just '.util') between client & consumer?